templates/partials/summary/collapsible-summary.html

```diff
@@ -3,44 +3,46 @@
 
 {%- set itemList = [] -%}
 
-{%- for group in content.summary.groups -%}
-  {%- if group["blocks"] | length -%}
-    {%- if group.title -%}
-      {%- set item = {
-        "title": group.title,
-        "id": group.id,
-        "content": ""
-      } -%}
+{%- for section, groups in content.summary.sections.items() -%}
```

Comment:
section isn't being used so this could be an iteration over the dictionary values, (or over the list as per point 1. on Mebins comment)

Reply:
Using a list now instead 👍